### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by JayHoltslander** [johngodley/redirection#671]
```
Plugin: 3.0.1
WordPress: 4.8.5
PHP: 7.0.27
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/604.5.6 (KHTML, like Gecko) Version/11.0.3 Safari/604.5.6

Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/404?filterBy=url-exact&filter=%2Fwp-updats.php DELETE
Code: 403 HTTP/2.0 403
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>403 Forbidden</title>
<style>
    body {margin: 20px; font-family: helvetica, sans-serif; max-width: 800px;}
    .error {color: #e00;}
    pre {font-size: 16px;}
    h1 {font-size: 28px;}
</style>
</head><body>
<h1>Forbidden</h1>
<p>You don't have permission to access /wp-json/redirection/v1/404
on this server.<br />
<br />
Possible causes of this error include:</p>
<ol>
<li>The request was forbidden by rules in the .htaccess file.</li>
<li>The directory you requested does not have an index.html or index.php file.</li>
<li>The permissions on the file or directory are incorrect.</li>
</ol>
<p class=error><b>For details about why the request was forbidden, see the Apache error log at:</b></p>
<pre class=error>/srv/users/SYSUSER/log/APPNAME/APPNAME_apache.error.log</pre>
<p><br /><br /><b>
</b></p>
</body></html>

```

**Comment by johngodley** [comment]
Please try the 3.1 beta:

https://github.com/johngodley/redirection/releases/tag/3.1

It should fix this problem

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
